compiler/rustc_codegen_llvm/src/gotoc/rvalue.rs

```diff
@@ -679,7 +679,7 @@ impl<'tcx> GotocCtx<'tcx> {
     ) -> Option<Expr> {
         // Check if the cast is from a vtable fat pointer to another
         // vtable fat pointer (which can happen with auto trait fat pointers)
-        if self.is_vtable_fat_pointer(src_mir_type) {
+        if self.is_ref_of_unsized(src_mir_type) {
```

Comment:
couldn't a ref of unsized be to a slice?

Reply:
Ah, I was thinking on this path it would have to be a dynamic, but you're right. Trying again!